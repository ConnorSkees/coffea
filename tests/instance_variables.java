public class instance_variables {

    public static final int a = 2;
    int b = this.a * 2;
    String c = "hiii";

    public void main(String[] args) {
        this.b = 3;
        String d = this.c;
    }
}
