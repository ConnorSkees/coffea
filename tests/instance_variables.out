public class instance_variables {
  public static final int a = 2;
  int b = (2 * 2);
  String c = "hiii";
  public void main(String[] arg1) {
    this.b = 3;
    String s2 = this.c;
    return;
  }
}
